apps/nextjs-website/react-components/components/Accordion/AccordionItem.tsx

```diff
@@ -32,14 +44,19 @@ export const AccordionItem: React.FC<AccordionItemProps> = ({
         targetItem.scrollIntoView({ behavior: 'smooth', block: 'center' });
       }
     }
-  }, [])
+  }, []);
+
+  const handleToggle = () => {
+    const newExpanded = !expanded;
+    setExpanded(newExpanded);
+    updateURL(itemID, newExpanded);
+  };
 
   return (
     <MUIAccordion
       disableGutters
-      // Removes MUI accordion's top border
       expanded={expanded}
-      onChange={() => setExpanded(!expanded)}
+      onChange={handleToggle}
```

Comment:
```suggestion
      onChange={handleChange}
```
Reflects previous suggestions